src/plugins/vis_type_vega/public/index.ts

```diff
@@ -35,3 +35,6 @@ import { VegaPlugin as Plugin } from './plugin';
 export function plugin(initializerContext: PluginInitializerContext<ConfigSchema>) {
   return new Plugin(initializerContext);
 }
+
+export { VegaExpressionFunctionDefinition } from './expressions/vega_fn';
+export { VegaSpecExpressionFunctionDefinition } from './expressions/line_vega_spec_fn';
```

Comment:
sticking with the `line` theme, maybe change to `LineVegaSpecExpressionFunctionDefinition`?